modules/bookCollection.js

```diff
@@ -0,0 +1,65 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] It would be nice if you remove this line 1 because linters help you fix those errors which can break your application 😨 So kindly remove this line and try to fix your linter error manually 👍 You can run the following command to fix this error ⬇️

```javascript
npx eslint . --fix
```